zio-sbt-ci/src/main/scala/zio/sbt/ZioSbtCiPlugin.scala

```diff
@@ -121,212 +139,264 @@ object ZioSbtCiPlugin extends AutoPlugin {
     val checkout            = Checkout.value
     val swapSizeGB          = ciSwapSizeGB.value
     val setSwapSpace        = SetSwapSpace.value
-    val javaVersion         = ciDefaultJavaVersion.value
     val checkGithubWorkflow = ciCheckGithubWorkflowSteps.value
-    val lint                = Lint.value
 
     Seq(
       Job(
-        id = "lint",
         name = "Lint",
         steps = (if (swapSizeGB > 0) Seq(setSwapSpace) else Seq.empty) ++
-          Seq(checkout, SetupLibuv, SetupJava(javaVersion), CacheDependencies) ++ checkGithubWorkflow ++ Seq(lint)
+          Seq(
+            checkout,
+            SetupLibuv,
+            SetupJava(ciDefaultJavaVersion.value, ciDefaultJavaDistribution.value),
+            CacheDependencies
+          ) ++ checkGithubWorkflow.flatMap(
+            _.flatten
+          ) ++
+          Seq(Lint.value) ++
+          (if (ciCheckMima.value) Seq(CheckMima.value) else Seq.empty[Step.SingleStep])
```

Comment:
Perhaps it is better to have dedicated mima jobs? Per Scala version/platform?

Reply:
checkMima is now part of the lint job. Can be disabled with `checkMima / skip := true`